projects/etos_suite_runner/src/etos_suite_runner/lib/executor.py

```diff
@@ -25,22 +28,19 @@ class Executor:  # pylint:disable=too-few-public-methods
 
     logger = logging.getLogger("ESR - Executor")
 
-    def __init__(self, etos):
+    def __init__(self, etos: ETOS) -> None:
```

Comment:
Should we annotate `__init__`, as an init cannot ever return anything other than `None`?

Reply:
```(Note that the return type of __init__ ought to be annotated with -> None. The reason for this is subtle. If __init__ assumed a return annotation of -> None, would that mean that an argument-less, un-annotated __init__ method should still be type-checked? Rather than leaving this ambiguous or introducing an exception to the exception, we simply say that __init__ ought to have a return annotation; the default behavior is thus the same as for other methods.)```

https://peps.python.org/pep-0484/#the-meaning-of-annotations